pkg/blob/bucket.go

```diff
@@ -59,6 +61,18 @@ func AWS(ctx context.Context, bucket string, conf *aws.Config) (*blob.Bucket, er
 	return s3blob.OpenBucket(ctx, s, bucket, nil)
 }
 
+func Azure(ctx context.Context, accountName string, containerName string) (*blob.Bucket, error) {
```

Comment:
Missing godoc

nit;

```suggestion
func Azure(ctx context.Context, accountName, containerName string) (*blob.Bucket, error) {
```

Reply:
Fixed